### ISSUE TITLE
Custom scheduling based on metadata

### ISSUE THREAD

**Issue by jnioche** [DigitalPebble/storm-crawler#283]
The default scheduler determines when a page should be re-fetched based on its status only and the same value is used regardless of the nature of a page e.g. RSS feed, sitemap, content page etc...

We could add a basic mechanism where a page could use an arbitrary key value in its metadata to set its nextFetchDate when it has been successfully fetched.

**Comment by jnioche** [comment]
This could be used initially for #291 so that feed URLs get refetched more frequently than the pages they point to

**Comment by jnioche** [comment]
Implemented in [e3ed3bb]

```
# custom fetch interval to be used when a document has the key/value in its metadata
# and has been fetched succesfully (value in minutes)
# fetchInterval.isFeed=true: 10
```
